app/src/main/java/com/duckduckgo/app/browser/BrowserWebViewClient.kt

```diff
@@ -424,12 +415,11 @@ class BrowserWebViewClient @Inject constructor(
             // See https://app.asana.com/0/0/1206159443951489/f (WebView limitations)
             if (it != ABOUT_BLANK && start == null) {
                 start = currentTimeProvider.elapsedRealtime()
-                maliciousSiteProtectionWebViewIntegration.onPageLoadStarted()
+                requestInterceptor.onPageStarted(url)
```

Comment:
I need to check if moving `requestInterceptor.onPageStarted(url)` here has any unintended side-effect. This is because onPageStarted can be called several times when a page is loaded